core/server/master/src/main/java/alluxio/master/backup/AbstractBackupRole.java

```diff
@@ -186,6 +186,8 @@ protected AlluxioURI takeBackup(BackupPRequest request, AtomicLong entryCounter)
           // Create the backup from master state.
           mBackupManager.backup(ufsStream, entryCounter);
         }
+        // Add a marker file indicating the file is completed.
+        ufs.create(backupFilePath + ".complete").close();
```

Comment:
Does anything look for this file?

Reply:
No it is not checked programatically.
Ideally we should check this when doing a restore, but that can break backward compatibility with backups taken by older versions.